app/src/pages/playground/constants.tsx

```diff
@@ -31,6 +31,7 @@ export const modelProviderToModelPrefixMap: Record<ModelProvider, string[]> = {
   GOOGLE: ["gemini"],
   DEEPSEEK: ["deepseek"],
   XAI: ["grok"],
+  OLLAMA: ["llama", "mistral", "codellama", "phi", "qwen", "gemma"],
```

Comment:
this is not going to be true long term - e.x. llama can come from bedrock, azure, etc., can you remove these?